iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -1513,7 +1513,7 @@ def add_polls_from_polls_index_page_to_footer_index_page_as_page_link_page(self)
         self.poll_index_page.refresh_from_db()
         self.footer_index_page.refresh_from_db()
         svg_title = 'clip board pen'
-        file = File(open(Path(settings.BASE_DIR) / 'iogt/static/icons/clip_board_pen.svg'), name=svg_title)
+        file = File(open(Path(settings.BASE_DIR) / 'iogt/static/icons/clip_board_pen.svg'), name='clip_board_pen.svg')
```

Comment:
```
file = File(open(Path(settings.BASE_DIR) / 'iogt/static/icons/clip_board_pen.svg'), name='clip_board_pen.svg')
icon = Svg.objects.create(title='clip board pen', file=file)
```

I feel this would be less confusing